crates/logger/src/logger.rs

```diff
@@ -59,8 +65,40 @@ impl Logger {
                     message
                 ));
             })
-            .chain(Dispatch::new().level(level).chain(io::stderr()))
-            .apply()
-            .unwrap();
+            .chain(io::stdout());
+
+        if let Some(output) = output {
+            let file_logger = Dispatch::new()
+                .chain(base_logger.clone())
+                .format(|out, message, record| {
+                    let formatted_timestamp = Local::now().format("%Y-%m-%d %H:%M:%S");
+
+                    let prefix = format!("[{} {}]", record.level(), formatted_timestamp);
+
+                    let formatted_message = format!("{} {} {}", prefix, record.target(), message);
+
+                    let message_without_colors_bytes = strip_ansi_escapes::strip(formatted_message)
+                        .expect("could not strip colors from log");
+
+                    let message_without_colors = std::str::from_utf8(&message_without_colors_bytes)
+                        .expect("could not decode de-colored log bytes into a str");
+
+                    out.finish(format_args!("{}", message_without_colors))
+                })
+                .chain(log_file(output).expect("yikes"));
+
+            Dispatch::new()
+                .level(level)
```

Comment:
My original thinking was that `--logFile` would override console logging entirely.

I'm not sure how other tools go about this, will need to look into.